src/OpenDebugAD7/DebugEventLogger.cs

```diff
@@ -114,40 +118,55 @@ public void Write(LoggingCategory category, string message, Dictionary<string, o
         {
             // Default to logging the message if we haven't set a status for this category.
             if (_isLoggingEnabled.ContainsKey(category) && !_isLoggingEnabled[category])
+            {
                 return;
+            }
 
             _sendToOutput.Invoke(CreateOutputEvent(category, message, data));
         }
 
-        /// <summary>
-        /// Sends the message to the raw output callback if the given category has logging enabled.
-        /// </summary>
-        /// <param name="category">The category of debug event logging.</param>
-        /// <param name="message">The message to log.</param>
-        /// <param name="data">Logging data to send alongside the message, if any.</param>
-        public void WriteRaw(LoggingCategory category, string message, Dictionary<string, object> data = null)
+        public void TraceLogger_EventHandler(object sender, LogEventArgs args)
+        {
+            if (args.Message.Contains("<--  "))
```

Comment:
>Contains [](start = 29, length = 8)

Should this be 'StartsWith'? Or do you really mean for it to be anywhere in the message string?